smart-cart/config/routes.rb

```diff
@@ -9,7 +9,7 @@
   get "lists", to: "lists#show"
   get "stores", to: "stores#show"
   get "instructions", to: "instructions#show"
-  post "add-item", to: "lists#add-item"
+  post "add-item", to: "lists#add-item", as: "add_item"
```

Comment:
```suggestion
  post "add-item", to: "lists#add-item", as: "add_item" # TODO: Delete this path if not used + update figma
```

Reply:
removed the route and updated figma!! gonna merge this now